### PR TITLE
Refactor/UI ux enhancement p5 merge dev 2405301308

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/utils/number-utils.spec.ts [hunk 1]
```diff
@@ -33,10 +33,10 @@ describe("mathSybmolAbsFormat", () => {
...
 
 describe("The toUnitFormat function returns a value represented in numeric units and up to two decimal places.", () => {
-  test("12345.6789 to $12.35K", () => {
-    const num = 12345.6789;
-    expect(toUnitFormat(num, true, true)).toBe("$12.35K");
-  });
+  // test("12345.6789 to $12.35K", () => {
+  //   const num = 12345.6789;
+  //   expect(toUnitFormat(num, true, true)).toBe("$12.35K");
+  // });
```
Comment (jinoosss) [inline packages/web/src/utils/number-utils.spec.ts:39]: Fix the results of the test case.

**Review by jinoosss — DISMISSED** [review]
LGTM !

File: packages/web/src/components/common/bar-graph/BarGraph.tsx [hunk 2]
```diff
@@ -183,6 +198,7 @@ const BarGraph: React.FC<BarGraphProps> = ({
     }
     const { currentTarget } = event;
     const { left, top } = currentTarget.getBoundingClientRect();
+    // const { left, top } = currentTarget.getBoundingClientRect();
```
Comment (jinoosss) [inline packages/web/src/components/common/bar-graph/BarGraph.tsx:201]: This is unused code.

File: packages/web/src/repositories/token/token-repository-mock.ts [hunk 3]
```diff
@@ -14,16 +14,23 @@ import { StorageClient } from "@common/clients/storage-client";
...
+import { TokenPriceModel } from "@models/token/token-price-model";
 
 export class TokenRepositoryMock implements TokenRepository {
   private localStorageClient: StorageClient<StorageKeyType>;
 
   constructor(localStorageClient: StorageClient) {
     this.localStorageClient = localStorageClient;
   }
+  public getTokenPricesByPath = async (path: string): Promise<TokenPriceModel> => {
+    console.log(path);
```
Comment (jinoosss) [inline packages/web/src/repositories/token/token-repository-mock.ts:30]: Clear the log.

File: packages/web/src/utils/number-utils.ts [hunk 4]
```diff
@@ -110,8 +110,12 @@ export const toUnitFormat = (
       unitsUpperCase.million
     );
   if (isKMB) {
-
     if (wholeNumberLength >= 4 && isFormat) {
+      console.log(bigNumber.toNumber().toString());
+      console.log(convertToKMB(bigNumber.toNumber().toString(), {
```
Comment (jinoosss) [inline packages/web/src/utils/number-utils.ts:115]: Clear the log.

File: packages/web/src/utils/number-utils.spec.ts (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/utils/number-utils.spec.ts:39]: I just comment it out fỏ testing , repopened it.

**Review by jinoosss — APPROVED** [review]

**Pull request merged**